src/services/symbolication.rs

```diff
@@ -1704,6 +1727,53 @@ impl SymbolicationActor {
                         });
                     }
 
+                    // If we successfully timed the new stackwalking method earlier, compute its
+                    // stacktraces as well
+                    if let Some((process_state_new, timer_new)) = process_state_and_timer_new {
+                    let requesting_thread_index: Option<usize> =
+                        process_state_new.requesting_thread().try_into().ok();
+                    let threads = process_state_new.threads();
+                    let mut stacktraces_new = Vec::with_capacity(threads.len());
+                    for (index, thread) in threads.iter().enumerate() {
+                        let registers = match thread.frames().get(0) {
+                            Some(frame) => map_symbolic_registers(
+                                frame.registers(minidump_state.system_info.cpu_arch),
+                            ),
+                            None => Registers::new(),
+                        };
+
+                        let frame_count = thread.frames().len().min(20000);
+                        let mut frames = Vec::with_capacity(frame_count);
+                        for frame in thread.frames().iter().take(frame_count) {
+                            let return_address =
+                                frame.return_address(minidump_state.system_info.cpu_arch);
+
+                            frames.push(RawFrame {
+                                instruction_addr: HexValue(return_address),
+                                package: frame.module().map(CodeModule::code_file),
+                                trust: frame.trust(),
+                                ..RawFrame::default()
+                            });
+                        }
+
+                        stacktraces_new.push(RawStacktrace {
+                            is_requesting: requesting_thread_index.map(|r| r == index),
+                            thread_id: Some(thread.thread_id().into()),
+                            registers,
+                            frames,
+                        });
+                    }
+
+                    // If both stacktraces are equal, report how long each method took, otherwise
+                    // report the fact that they differed.
+                    if stacktraces == stacktraces_new {
+                        metric!(timer("minidump.stackwalk.duration") = timer_old, "method" => "old");
+                        metric!(timer("minidump.stackwalk.duration") = timer_new, "method" => "new");
+                    } else {
+                        metric!(counter("minidump.stackwalk.differences") += 1);
```

Comment:
I think we need two counters, one for the case where there are differences and one for the case where they are the same.  That will allow easy checking what ratio has differences.
Either as two metrics or as a single metric with different tags, I'm not sure which is easier in datadog, might not make much difference in practice.